### DISCUSSION TITLE
Feature Row for Splash Page won't wokr

### DISCUSSION THREAD

**Discussion by aymanzyy** [mmistakes/minimal-mistakes#3344] [Q&A]
I was trying to implement the splash page but was having trouble getting my images to show. Instead I just get a blank square. Anyone know a fix?

![image](https://user-images.githubusercontent.com/66223526/149645198-0de3e1b5-a83c-4a0a-8e4d-65aaa49293ce.png)

**Comment by iBug** [comment]
We won't know what's going wrong until you share some code or your website so we have something to examine or inspect.
